repl/repl.go

```diff
@@ -0,0 +1,114 @@
+// Package repl implements a REPL (read-eval-print loop) for otto.
+package repl
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"gopkg.in/readline.v1"
+)
+
+var i int
+
+// DebuggerHandler implements otto's debugger handler signature, providing a
+// simple drop-in debugger implementation.
+func DebuggerHandler(vm *otto.Otto) {
+	i++
```

Comment:
Not sure how often people will be running this in a multi-threaded context, but this doesn't look thread-safe. Maybe use atomic.AddUint32 here.